[
    {
        "author_association": "CONTRIBUTOR",
        "body": "Just wanna note the only test that seems to have failed, did so because SQLite attempted to operate on a readonly database? ",
        "created_at": "2021-01-09T11:06:46Z",
        "html_url": "https://github.com/ipython/ipython/pull/12739#issuecomment-757133290",
        "id": 757133290,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12739",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1NzEzMzI5MA==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/757133290/reactions"
        },
        "updated_at": "2021-01-09T11:06:46Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/757133290",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/20028782?v=4",
            "events_url": "https://api.github.com/users/farisachugthai/events{/privacy}",
            "followers_url": "https://api.github.com/users/farisachugthai/followers",
            "following_url": "https://api.github.com/users/farisachugthai/following{/other_user}",
            "gists_url": "https://api.github.com/users/farisachugthai/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/farisachugthai",
            "id": 20028782,
            "login": "farisachugthai",
            "node_id": "MDQ6VXNlcjIwMDI4Nzgy",
            "organizations_url": "https://api.github.com/users/farisachugthai/orgs",
            "received_events_url": "https://api.github.com/users/farisachugthai/received_events",
            "repos_url": "https://api.github.com/users/farisachugthai/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/farisachugthai/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/farisachugthai/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/farisachugthai"
        }
    },
    {
        "author_association": "MEMBER",
        "body": "> Just wanna note the only test that seems to have failed, did so because SQLite attempted to operate on a readonly database?\r\n\r\nthat's unrelated, it's a race condition on travis. Thanks.",
        "created_at": "2021-01-11T16:29:19Z",
        "html_url": "https://github.com/ipython/ipython/pull/12739#issuecomment-758068436",
        "id": 758068436,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12739",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1ODA2ODQzNg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/758068436/reactions"
        },
        "updated_at": "2021-01-11T16:29:19Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/758068436",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/335567?v=4",
            "events_url": "https://api.github.com/users/Carreau/events{/privacy}",
            "followers_url": "https://api.github.com/users/Carreau/followers",
            "following_url": "https://api.github.com/users/Carreau/following{/other_user}",
            "gists_url": "https://api.github.com/users/Carreau/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/Carreau",
            "id": 335567,
            "login": "Carreau",
            "node_id": "MDQ6VXNlcjMzNTU2Nw==",
            "organizations_url": "https://api.github.com/users/Carreau/orgs",
            "received_events_url": "https://api.github.com/users/Carreau/received_events",
            "repos_url": "https://api.github.com/users/Carreau/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/Carreau/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/Carreau/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/Carreau"
        }
    },
    {
        "author_association": "CONTRIBUTOR",
        "body": "Owee, I'm MrMeeseeks, Look at me.\n\nThere seem to be a conflict, please backport manually. Here are approximate instructions:\n\n1. Checkout backport branch and update it.\n\n```\n$ git checkout 7.x\n$ git pull\n```\n\n2. Cherry pick the first parent branch of the this PR on top of the older branch:\n```\n$ git cherry-pick -m1 6a7c488093026577db0d5c5f21d2aea129e5b6c9\n```\n\n3. You will likely have some merge/cherry-pick conflict here, fix them and commit:\n\n```\n$ git commit -am \"Backport PR #12739: BUG: The `%conda` magic doesn't work\"\n```\n\n4. Push to a named branch :\n\n```\ngit push YOURFORK 7.x:auto-backport-of-pr-12739-on-7.x\n```\n\n5. Create a PR against branch 7.x, I would have named this PR:\n\n> \"Backport PR #12739 on branch 7.x\"\n\nAnd apply the correct labels and milestones.\n\nCongratulation you did some good work ! Hopefully your backport PR will be tested by the continuous integration and merged soon!\n\nIf these instruction are inaccurate, feel free to [suggest an improvement](https://github.com/MeeseeksBox/MeeseeksDev).\n                ",
        "created_at": "2021-01-11T16:29:41Z",
        "html_url": "https://github.com/ipython/ipython/pull/12739#issuecomment-758068672",
        "id": 758068672,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12739",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1ODA2ODY3Mg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/758068672/reactions"
        },
        "updated_at": "2021-01-11T16:29:41Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/758068672",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/in/812?v=4",
            "events_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/events{/privacy}",
            "followers_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/followers",
            "following_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/following{/other_user}",
            "gists_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/apps/lumberbot-app",
            "id": 24485218,
            "login": "lumberbot-app[bot]",
            "node_id": "MDM6Qm90MjQ0ODUyMTg=",
            "organizations_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/orgs",
            "received_events_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/received_events",
            "repos_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/lumberbot-app%5Bbot%5D/subscriptions",
            "type": "Bot",
            "url": "https://api.github.com/users/lumberbot-app%5Bbot%5D"
        }
    }
]